tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java

```diff
@@ -52,6 +54,11 @@
    */
   private List<MeasurementSchema> schemas;
 
+  /**
+   * measurementId->indexOf(measurementSchema)
+   */
+  private Map<String, Integer> mschemawithindex;
```

Comment:
mschemawithindex should be camel-case name, like mSchemaWithIndex.

Reply:
copy that.